bash_completion

```diff
@@ -245,6 +245,56 @@ _upvars()
     done
 }
 
+# Get the list of filenames that match with the specified glob pattern.
+# @param $1 array_name  Array name
+#   The array name should not start with the double underscores "__".  The
+#   array name should not be "GLOBIGNORE".
+# @param $2 pattern     Pattern string to be evaluated.
+#   This pattern string will be evaluated using "eval", so brace expansions,
+#   parameter expansions, command substitutions, and other expansinos will be
+#   processed.  The user-provided strings should not be directly specified to
+#   this argument.
+_comp_expand_glob()
+{
+    if (($# != 2)); then
+        printf '%s\n' "bash-completion: $FUNCNAME: unexpected number of arguments." >&2
```

Comment:
Thank you for the review! I will address them one by one.

> just simply `echo`,

Simple `echo` has a problem with `shopt -s xpg_echo` being turned on, so I don't want to use `echo` with parameter expansions. But maybe it is fine for the present case because `$FUNCNAME` will not contain backslashes unless the user unsets `FUNCNAME` and rewrites it. I will just apply the suggested change.

Reply:
Updated in 9f3aae24